apps/teams-test-app/src/components/VisualMediaAPIs.tsx

```diff
@@ -0,0 +1,104 @@
+import { visualMedia } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const imagesHelper = (medias: visualMedia.VisualMediaFile[]): string => {
```

Comment:
suggest to change API name from imagesHelper to imagesMessageBuilder (something like that) to make it easier to be understood.

Reply:
Great suggestion. Updated.